server/reload.go

```diff
@@ -1589,6 +1589,8 @@ func (s *Server) diffOptions(newOpts *Options) ([]option, error) {
 			diffOpts = append(diffOpts, &ocspOption{newValue: newValue.(*OCSPConfig)})
 		case "ocspcacheconfig":
 			diffOpts = append(diffOpts, &ocspResponseCacheOption{newValue: newValue.(*OCSPResponseCacheConfig)})
+		case "configdigest":
```

Comment:
Bit unclear what this is for, aren't we supposed to be populating `ConfigDigest` automatically in `ProcessConfigFile`? Is there a reason why we'd want to take the digest and override it from the config input?

Reply:
good point, updated to skip it since already done as you mention. I just have to add something here since otherwise it would complain with `config reload not supported` for field but a `continue` would suffice.